### DISCUSSION TITLE
Is there a way to temporarily set keyboard controls to the navigation drawer only?

### DISCUSSION THREAD

**Discussion by tjeb** [vuetifyjs/vuetify#14743] [Q&A]
Hi,

I have a navigation drawer that is permanently visible on large screens, and can be toggled on and off on small screens. In order to comply with WCAG requirements, I need keyboard tab-control to remain in the navigation drawer when it is shown on small screens (so that the toolbar and the page elements are not focusable). However, right now, tab-key navigation works the same as for large screens (i.e. when pressing tab, focus goes to toolbar first, then navigation drawer, then rest of page).

Is there any way to toggle keyboard controls of the toolbar and the rest of the page, when the navigation drawer is opened/closed?

**Comment by KaelWD** [comment]
Not built in but you could copy it from VDialog: 

https://github.com/vuetifyjs/vuetify/blob/860be6b0bff74b6cd81320259879ae9dc5a77f1d/packages/vuetify/src/components/VDialog/VDialog.ts#L189-L194
https://github.com/vuetifyjs/vuetify/blob/860be6b0bff74b6cd81320259879ae9dc5a77f1d/packages/vuetify/src/components/VDialog/VDialog.ts#L218-L240

This probably should be added to temporary drawers by default though.
